### PR TITLE
phone number combination

### PR DESCRIPTION
**Pull request by noelfoka**
Here, I added the Letter Combination of a Phone Number in the Algorithm Challenge readme file

### CODE REVIEW

**Comment by dassiorleando** [comment]
Looks like you had trouble to continue using the other PR, I'm closing #33 then.

**Review by messaismael — CHANGES_REQUESTED** [review]
I don't think you should delete this file  @noelfoka 
@dassiorleando

File: algorithms/phone-number.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## Context&nbsp;[![Hits](https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2Fnumerica-ideas%2Fcommunity%2Ftree%2Fmaster%2Falgorithms%2Fnumber-of-zero-filled-subarrays&count_bg=%2379C83D&title_bg=%23555555&icon=&icon_color=%23E7E7E7&title=hits&edge_flat=false)](https://numericaideas.com/)
-This challenge comes from Leetcode, it's about Letter Combinations of a Phone Number.
+This challenge comes from Leetcode,.......it's about finding all possible letter combinations that the provided number could represent on a cellular phone keyboard.
```
Comment (messaismael) [inline algorithms/phone-number.md:2]: Remove the (........).

File: algorithms/phone-number.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  function combinaison(numeroEntree: string, listCombPrec: string) {
+    let accComb: string[] = [];
+    for(let combPrec of listComprec) {
+      for(let combSuiv of list[numeroEntree[0]]) {
+        accComb.push(combPrec.concat(combSuiv));
+      }
+    }
+    return numeroEntree.length > 1 ?combinaison(numeroentree.substring(1), accComb) :accComb;
+  }
+  return !digits.length ?[] :combinaison(digits, ['']);
```
Comment (messaismael) [inline algorithms/phone-number.md:32]: `return !digits.length ? [] : combinaison(digits, ['']);`

File: algorithms/README.md [hunk 3]
```diff
@@ -5,6 +5,7 @@
 Here are some accepted solutions from our challenges:
 
 - [Number of Zero-Filled Subarrays](./number-of-zero-filled-subarrays.md)
+- [Letter Combination of a Phone Number](./phone-number.md)
```
Comment (messaismael) [inline algorithms/README.md:8]: Change the

`(./phone-number.md)`

by 

`(./number-of-zero-filled-subarrays.md)`

**Review by messaismael — CHANGES_REQUESTED** [review]

File: algorithms/letter-combinations-of-a-phone-number.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-## Context&nbsp;[![Hits](https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2Fnumerica-ideas%2Fcommunity%2Ftree%2Fmaster%2Falgorithms%2Fphone-number&count_bg=%2379C83D&title_bg=%23555555&icon=&icon_color=%23E7E7E7&title=hits&edge_flat=false)](https://numericaideas.com/)
+## Context&nbsp;[![Hits](https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2Fnumerica-ideas%2Fcommunity%2Ftree%2Fmaster%2Falgorithms%2Fletter-combinations-of-a-phone-number.md&count_bg=%2379C83D&title_bg=%23555555&icon=&icon_color=%23E7E7E7&title=hits&edge_flat=false)](https://numericaideas.com/)
```
Comment (messaismael) [inline algorithms/letter-combinations-of-a-phone-number.md:1]: Don't add the `md` extension here.

**Review by messaismael — CHANGES_REQUESTED** [review]

**Review by messaismael — APPROVED** [review]

**Comment by messaismael** [comment]
@dassiorleando It's ok

**Review by dassiorleando — APPROVED** [review]

**Pull request merged**